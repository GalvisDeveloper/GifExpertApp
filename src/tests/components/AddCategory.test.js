

import React from 'react';
import { shallow } from 'enzyme';
import "@testing-library/jest-dom";

describe('Testeando el componente AddCategory', () => {
    test('should ', () => {

    })
});
