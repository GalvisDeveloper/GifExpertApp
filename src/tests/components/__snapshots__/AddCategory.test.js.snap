// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testeando el componente AddCategory Prueba 1: Coincidir con el snapshot  1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    placeholder="Type here the category of gif that you want"
    type="text"
    value=""
  />
</form>
`;
